apps/site/lib/site_web/templates/schedule/_timetable.html.eex

```diff
@@ -1,4 +1,22 @@
 <%= SiteWeb.AlertView.group(alerts: @alerts, route: @route, date_time: @date_time, priority_filter: :high) %>
+<%= if @route.id == "Orange" do %>
+  <p>
+    During this shutdown, riders can simply show their CharlieCard or CharlieTicket to conductors to ride the Commuter Rail in
+    <a target="_blank" href="https://cdn.mbta.com/sites/default/files/2021-03/2021-03-23-cr-fare-zones.pdf">Zones 1A, 1 and 2 </a>at no charge.
+  </p>
+  <h2>North Side Alternatives</h2>
+  <p>
+    For service on the north side of the Orange Line, most
+    <a target="_blank" href="https://www.mbta.com/schedules/CR-Haverhill/timetable">Haverhill Line</a> trains will stop at
+    <a target="_blank" href="https://www.mbta.com/stops/place-ogmnl">Oak Grove</a> en route to
+    <a target="_blank" href="https://www.mbta.com/stops/place-mlmnl">Malden Center</a> and
+    <a target="_blank" href="https://www.mbta.com/stops/place-north">North Station</a>
```

Comment:
nitpick: use the url helpers  Phoenix provides, e.g. `href="<%= stop_path(@conn, :show, "place-north") %>"`